relay-profiling/src/sample/v2.rs

```diff
@@ -33,6 +33,8 @@ pub struct ProfileMetadata {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub environment: Option<String>,
     pub platform: String,
+
+    #[serde(default)]
     pub release: String,
```

Comment:
Feel free to not do this, might be too much to change, but an `Option` would be preferable, it's implicitly `#[serde(default)]`.

```suggestion
    pub release: Option<String>,
```